### ISSUE TITLE
Some imperfections in documentation

### ISSUE THREAD

**Issue by qin-yu** [computational-cell-analytics/micro-sam#355]
1. https://computational-cell-analytics.github.io/micro-sam/micro_sam.html#from-source `cd micro_sam` to `cd micro-sam`
2. https://computational-cell-analytics.github.io/micro-sam/micro_sam.html#how-to-use-the-python-library I don't think the modules are exposed after `import micro_sam`
3. https://computational-cell-analytics.github.io/micro-sam/micro_sam.html#tips-tricks `micro_sam.precompute_state` to `micro_sam.precompute_embeddings`

**Comment by anwai98** [comment]
Hi @qin-yu,

Thanks for the mentions. I updated `1.` and `3.` at https://github.com/computational-cell-analytics/micro-sam/pull/357.

For `2.`, you can use the submodules in python scripts similar to the mentions here: [micro_sam.training](https://github.com/computational-cell-analytics/micro-sam/blob/master/examples/finetuning/finetune_hela.py#L11), [micro_sam.instance_segmentation](https://github.com/computational-cell-analytics/micro-sam/blob/master/examples/finetuning/instance_segmentation_with_finetuned_model.py#L4-L6).

**Comment by constantinpape** [comment]
Fixed by #357

And re 2. : @anwai98 is correct, you can import these components via
```python
import micro_sam.training
```
or
```python
import micro_sam.training as sam_training
```

I will update the doc in #359 to make this clear.
